frame/offences/benchmarking/src/lib.rs

```diff
@@ -315,13 +315,13 @@ benchmarks! {
 			<T as StakingConfig>::Event::from(StakingEvent::<T>::Slashed(id, BalanceOf::<T>::from(slash_amount)))
 		);
 		let balance_slash = |id| core::iter::once(
-			<T as BalancesConfig>::Event::from(pallet_balances::Event::<T>::Slashed(id, slash_amount.into()))
+			<T as BalancesConfig>::Event::from(pallet_balances::Event::<T>::Slashed{who: id, amount: slash_amount.into()})
```

Comment:
```suggestion
			<T as BalancesConfig>::Event::from(pallet_balances::Event::<T>::Slashed{ who: id, amount: slash_amount.into() })
```

Same for the rest of the file